src/main/java/uk/gov/companieshouse/missingimagedelivery/orders/api/model/FilingHistoryCategory.java

```diff
@@ -0,0 +1,203 @@
+package uk.gov.companieshouse.missingimagedelivery.orders.api.model;
+
+import uk.gov.companieshouse.missingimagedelivery.orders.api.converter.EnumValueNameConverter;
+
+public enum FilingHistoryCategory {
+    ACCOUNTS() {
```

Comment:
Oh goodness, that's a lot of enums! It might be easier to do this?
```
public enum FilingHistoryCategory {
    ACCOUNTS(ProductType.MISSING_IMAGE_DELIVERY_ACCOUNTS),
    ANNUAL_RETURN(ProductType.MISSING_IMAGE_DELIVERY_ANNUAL_RETURN)
    ....


    private final ProductType productType;

    private FilingHistoryCategory(ProductType productType) {
        this.productType = productType;
    }
}
```

Reply:
Good point. 👍